dmoj/cptbox/isolate.py

```diff
@@ -427,6 +427,8 @@ def __init__(self, handler: Callable, reason: str):
         self.reason = reason
 
     def log(self, syscall: int):
-        syscall_name = by_id[syscall].lstrip('sys_')
+        syscall_name = by_id[syscall]
+        assert syscall_name.startswith('sys_')
```

Comment:
Is this really worth failing over? How about just if it starts with `sys_` do `[:4]`?

Reply:
`.replace('sys_', '', 1)`?

Reply:
I think I'd rather be a bit more explicit and take quantum's suggestion.